wordpress-mobile/WordPress-iOS
# Update swift lint

This PR updates to latest swiflint version and does a run with auto-correct to fix any current violations of rules.

Majority of fails are incorrect spacing after colons thing like this: `let a:String ="Hello"`

To test:
 - Make sure project build
 - Make sure tests build

Needs review: @koke @frosty mind to give it a look.
